content/en/api/payments.md

```diff
@@ -5,4 +5,99 @@ position: 41
 category: API
 ---
 
-Payments
\ No newline at end of file
+## Capture
+
+The capture endpoint allows capturing funds from a Single Payment authorisation, a Frequent Payment or a Frequent Payment authorisation. It also allows getting the details of a previous capture.
+
```

Comment:
I believe it's a Frequent Payment "from scratch" rather than a previous authorisation. The original text is here: https://api.prod.easypay.pt/docs#tag/Payment-Generic-Operations/paths/~1capture~1%7Bid%7D/get

Reply:
OK then.